addons/zeus/XEH_postInit.sqf

```diff
@@ -2,7 +2,7 @@
 
 ["ace_settingsInitialized",{
     // Only add an InitPost EH if setting is enabled (and apply retroactively)
-    if (isServer && GVAR(autoAddObjects)) then {
+    if (isServer && {GVAR(autoAddObjects)}) then {
```

Comment:
It's always slower on the server, but faster on the clients. Since there are more clients than servers usually, this is overall faster.
Not that it matters.